addons/udes_stock/models/stock_move_line.py

```diff
@@ -230,3 +233,15 @@ def _split(self, uom_qty=None):
             )
             res = new_ml
         return res
+
+    def get_fast_move_lines(self, domain, aux_domain=None):
```

Comment:
Not sure about the name of this function, is it just a faster (less cpu cycles) way to get stock.move.line recordsets with a search domain?

Could be worth putting on basemodel as `def search_orderby_id()` as it seems like a pretty generic function

Reply:
It is Miquel's suggestion name, putting into a method etc :)

Reply:
Could definitely be useful on basemodel, but I can raise an issue for that.

This name is a bit misleading though, as we have product speeds etc.

Tbh just `get_move_lines` would do... with a comment in the doc-string about performance increase as Peter A suggested.

Reply:
As Pete suggests, let's call this function `get_move_lines` with an option `order` parameter. Everywhere we currently call `get_fast_move_lines` passing in "id" as the order. This should make it very clear as to what the function's doing.

Reply:
get_move_lines exists like a method with optional done , which filters done moves or incomplete moves